config.template.json

```
@@ -36,7 +36,8 @@
             "external_url_prefix": ""
         },
         "mraid_support": false,
-        "snapchat_cta": false,
+        "cta_option": false,
+        "adOrientation": false,
```

```suggestion
        "ad_orientation": false,
```
To keep inline with naming of other properties

---

I saw that a few minutes after I did the commit, and figured this feedback would appear. Will update!